bin/format.dart

```diff
@@ -170,14 +173,19 @@ void main(List<String> args) {
     }
   }
 
+  var stdinName = argResults["stdin-name"];
```

Comment:
[nit] consider adding a type on the left, or making it explicit with `as String` on the right. As is the implicit cast happens on line 188 and if it fails the error would be nicer here.

Reply:
Done.